tests/test_h2o_benchmarks.py

```diff
@@ -3,105 +3,123 @@
 """
 
 import dask.dataframe as dd
-import pandas as pd
 import pytest
 
+# import pandas as pd
+
 
 @pytest.fixture(
     scope="module",
     params=[
         "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_single.csv",
         # "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_single.csv",
         # "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_single.csv",
+        "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_parquet/*.parquet",
+        # "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_parquet/*.parquet",
+        # "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_parquet/*.parquet",
     ],
     ids=[
-        "0.5 GB",
-        # "5 GB",
-        # "50 GB",
+        "0.5 GB (csv)",
+        # "5 GB (csv)",
+        # "50 GB (csv)",
+        "0.5 GB (parquet)",
+        # "5 GB (parquet)",
+        # "50 GB (parquet)",
     ],
 )
 def ddf(request):
-    yield dd.read_csv(
-        request.param,
-        dtype={
-            "id1": "category",
-            "id2": "category",
-            "id3": "category",
-            "id4": "Int32",
-            "id5": "Int32",
-            "id6": "Int32",
-            "v1": "Int32",
-            "v2": "Int32",
-            "v3": "float64",
-        },
-        storage_options={"anon": True},
-    )
+
+    if request.param.endswith("csv"):
+        yield dd.read_csv(
+            request.param,
+            dtype={
+                "id1": "category",
+                "id2": "category",
+                "id3": "category",
+                "id4": "Int32",
+                "id5": "Int32",
+                "id6": "Int32",
+                "v1": "Int32",
+                "v2": "Int32",
+                "v3": "float64",
+            },
+            storage_options={"anon": True},
+        )
+    elif request.param.endswith("parquet"):
```

Comment:
Since there's only two options (csv vs. parquet) we can just use `else` here 

```suggestion
    else:
```